package.json

```diff
@@ -4,15 +4,20 @@
   "description": "Javascript library for DHIS2",
   "main": "d2.js",
   "scripts": {
-    "documentation": "./node_modules/.bin/jsdoc -c jsdoc.json --readme ./README.md -u ./tutorials",
+    "babel": "babel src --out-dir build --source-maps --ignore '/**/__tests__/*','/**/__mocks__/*','/**/__fixtures__/*'",
```

Comment:
Don't know if you know what this is for, but would be nice if the name for the script was describing what's being done when the command is run.

Reply:
Hmm yeah basically it's transpiling the js to es5 so it can be distributed via npm.

Reply:
I see, so maybe calling this one `transpile-to-es5` makes more sense here?

Reply:
Would you agree with `transpile`? I think that'd convey what's happening there right?

Reply:
absolutely

Reply:
Cool, I'll add it 👍